### PR TITLE
CLN: xarray tests

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/generic/test_to_xarray.py [hunk 1]
```diff
@@ -106,19 +92,14 @@ def test_to_xarray(self):
...
-    )
+    @td.skip_if_no("xarray", "0.10.0")
     def test_to_xarray_index_types(self, indices):
         if isinstance(indices, pd.MultiIndex):
             pytest.skip("MultiIndex is tested separately")
 
         from xarray import DataArray
 
-        s = Series(range(len(indices)), index=indices)
+        s = Series(range(len(indices)), index=indices, dtype="float64")
```
Comment (jbrockmendel) [inline pandas/tests/generic/test_to_xarray.py]: why is this change needed?

Comment (MomIsBestFriend) [reply pandas/tests/generic/test_to_xarray.py]: To suppress the Deprecation warning

<details>

```DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.```

</details>

**Comment by jbrockmendel** [comment]
LGTM

File: pandas/tests/generic/test_to_xarray.py (hunk 1 above)
Comment (WillAyd) [reply pandas/tests/generic/test_to_xarray.py]: Is this the only place this occurs? I assume we are getting an empty container through from indices

I'd be surprised if this is the only case, so not sure doing as a one-off here

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/tests/generic/test_to_xarray.py (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/tests/generic/test_to_xarray.py]: > Is this the only place this occurs? I assume we are getting an empty container through from indices
> 
> I'd be surprised if this is the only case, so not sure doing as a one-off here

@WillAyd This is the only case in this file that raises the deprecation warning.

Comment (MomIsBestFriend) [reply pandas/tests/generic/test_to_xarray.py]: I can revert this change, not a deal breaker for me

**Review by simonjayhawkins — APPROVED** [review]
Thanks @MomIsBestFriend lgtm ex @WillAyd comment.

**Comment by simonjayhawkins** [comment]
Thanks @MomIsBestFriend
